### PR TITLE
Added wheelchair icon

### PR DESCRIPTION
**Pull request by mcellier**

### CODE REVIEW

File: src/icon/wheelchair.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react'
+import cc from 'classcat'
+import isEmpty from 'lodash.isempty'
+
+import { color } from '_utils/branding'
+
+class WheelchairIcon extends PureComponent<Icon> {
+  static defaultProps: Icon = {
```
Comment (grosbouddha) [inline src/icon/wheelchair.tsx]: Out of scope for this PR but it hurts to see so much duplicated code in the Icon code :/

Comment (Tuxosaurus) [reply src/icon/wheelchair.tsx]: Indeed...

File: src/icon/wheelchair.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        viewBox="0 0 100 100"
+        xmlns="http://www.w3.org/2000/svg"
+        className={cc(['kirk-icon', className])}
+        width={size}
+        height={size}
+        aria-hidden={isEmpty(title)}
+      >
+        {title && <title>{title}</title>}
+        <g fill={iconColor}>
+          <path
```
Comment (Tuxosaurus) [inline src/icon/wheelchair.tsx]: Looks like the paths can be further optimized: https://jakearchibald.github.io/svgomg/

Comment (mcellier) [reply src/icon/wheelchair.tsx]: Done. Thx.

File: src/icon/wheelchair.tsx (hunk 1 above)
Comment (mcellier) [reply src/icon/wheelchair.tsx]: #221

**Pull request merged**